provider/provider.go

```diff
@@ -130,6 +119,16 @@ func (p *Provider) Schema(ctx context.Context, req tfprovider.SchemaRequest, res
 	}
 }
 
+func (p *Provider) getValue(attr types.String, envSuffix string) (string, bool) {
```

Comment:
Maybe change the name? it's a bit ambiguous.

Reply:
As discussed, I changed the function name to `getAttrValueOrConfig`